packages/extension-ui/src/components/InputWithLabel.tsx

```diff
@@ -90,8 +90,11 @@ function InputWithLabel({
         value={value}
         withError={isError}
       />
-      {type === "password" && (
-        <IconButton onClick={toggleObscure}>
+      {type === 'password' && (
+        <IconButton
+          onClick={toggleObscure}
+          type='button'
```

Comment:
A `button` with `type="button"`?

Reply:
that's correct, all button that don't submit or reset form should have this set

Reply:
_[cross-linking for reference](https://github.com/Cardinal-Cryptography/aleph-zero-signer/pull/81#discussion_r1207045580)_